### ISSUE TITLE
Google analytics

### ISSUE THREAD

**Issue by grdw** [quintel/etcentral#166]
- Track page views
- Track which application is opened

**Comment by redekok** [comment]
@antw Do you know if we are yet tracking the things mentioned above?

**Comment by antw** [comment]
> Do you know if we are yet tracking the things mentioned above?

I do not believe so. Currently we keep track of (anonymised) data for the main ETM application only.

The basic setup has been done for ETCentral, but it'll need a few adjustments – and an updated config on the server – to enable it.
